plugins/channelrx/demoddab/dabdemodsink.cpp

```diff
@@ -488,7 +488,6 @@ void DABDemodSink::processOneAudioSample(Complex &ci)
 }
 
 DABDemodSink::DABDemodSink(DABDemod *packetDemod) :
-        m_dabDemod(packetDemod),
```

Comment:
Remove from constructor parameters if not needed

Reply:
ok